build.go

```diff
@@ -163,6 +165,11 @@ func GfmrunActionFunc(c *cli.Context) error {
 }
 
 func TocActionFunc(c *cli.Context) error {
+	if runtime.GOOS == "windows" {
```

Comment:
do you want to move this to build_windows.go ?

Reply:
I was trying to avoid adding a separate file 🤷🏼